gradlew

```diff
@@ -1,5 +1,21 @@
 #!/usr/bin/env sh
 
+#
```

Comment:
If we are going to add copyright headings it ought to be the correct one...

```
/*
 * Hibernate, Relational Persistence for Idiomatic Java
 *
 * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
 * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
 */
```

Reply:
If you want to leave it as Apache, that's fine too.  I just mean the style

Reply:
I did not add these copyright headers, the Gradle Wrapper plugin did, automatically, when I upgraded. It's code from the Gradle Wrapper plugin, and I do not feel comfortable about changing either the license or the copyright holders.

Reply:
Besides, this file will be replaced automatically each time we upgrade to a newer Gradle version.

Reply:
This is something new they have added then.  It is clearly new additions (new lines).  I just assumed you added them.  No problem then